libs/common-services/src/main/java/uk/gov/di/ipv/core/library/dto/CredentialIssuerConfig.java

```diff
@@ -17,8 +19,13 @@ public class CredentialIssuerConfig {
     private URI credentialUrl;
     private URI authorizeUrl;
     private String clientId;
+
+    @JsonDeserialize(using = KeepAsJsonDeserializer.class)
```

Comment:
I think this is an interesting question - should we embed the JWK JSON objects directly into the CRI config JSON, or should we double-encode it?

I'd probably have a slight preference towards the double-encoding if we're treating as an opaque string in our code, as it's simpler to implement and more representative of how we're using it (we don't expect to edit the key content ourselves).

Reply:
Makes sense - I'll go with the double encoding route and handle accordingly